### PR TITLE
Demo user

### PR DESCRIPTION
**Pull request by bnelson7**

### CODE REVIEW

File: frontend/src/components/navbar/navbar.js [hunk 1]
```diff
@@ -8,13 +8,20 @@ class NavBar extends React.Component {
...
     logoutUser(e) {
         e.preventDefault();
         this.props.logout();
     }
 
+
+    handleDemo(e) {
+        e.preventDefault();
+        this.props.loginDemoUser().then(() => this.props.history.push('/'))
+    }
```
Comment (haydenlinder) [inline frontend/src/components/navbar/navbar.js]: push on tickets/owner

Comment (bnelson7) [reply frontend/src/components/navbar/navbar.js]: but its a demo user

File: frontend/src/components/navbar/navbar_container.js [hunk 2]
```diff
@@ -8,7 +8,13 @@ const mapStateToProps = state => ({
...
 
+const mapDispatchToProps = dispatch => ({
+    logout: () => dispatch(logout()),
+    loginDemoUser: () => dispatch(loginDemoUser())
+})
+
 export default connect(
     mapStateToProps,
-    { logout }
+    // { logout }
```
Comment (haydenlinder) [inline frontend/src/components/navbar/navbar_container.js]: remove this comment?

Comment (bnelson7) [reply frontend/src/components/navbar/navbar_container.js]: ok

File: frontend/src/components/navbar/navbar.js (hunk 1 above)
Comment (haydenlinder) [reply frontend/src/components/navbar/navbar.js]: demo user will have tickets

**Pull request merged**